src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheckTest.java

```diff
@@ -93,7 +93,7 @@ public void testSameDoesNotComplainForNonMultilineConstructs() throws Exception
         final DefaultConfiguration checkConfig = createModuleConfig(RightCurlyCheck.class);
```

Comment:
Please double check and ensure the new token is tested on every type of option we have and each one generates an error.
I don't see `same` option producing a warning in the tests similar to https://sd1998.github.io/checkstyle-regression/checkstyle/index.html#A14 .
I would ensure we are testing alot more variations of annotation definitions with braces in different positions to make sure all cases are covered. That includes violation cases and non-violation cases.

Reply:
@sd1998 Is this done? I recommend replying to each item so we are sure we are on the same page. This item was a little open ended, so I want to make sure it is done before finalizing.

Reply:
@rnveach I have added cases for violations and non-violations for all the three options.